go-controller/pkg/metrics/ovs.go

```diff
@@ -487,9 +502,13 @@ func getOvsBridgeOpenFlowsCount(ovsOfctl ovsClient, bridgeName string) (float64,
 }
 
 func ovsInterfaceMetricsUpdater(ovsVsctl ovsClient, tickPeriod time.Duration, stopChan <-chan struct{}) {
+	stdout, err := getOvsInterfaces(ovsVsctl)
+	if err != nil {
+		klog.Errorf("Updating OVS interface metrics failed: %s", err.Error())
+	}
+	MetricOvsInterfaceTotal.Set(float64(len(strings.Split(stdout, "\n"))))
```

Comment:
this will only update once and never again right? shouldnt it be periodically updated?